packages/notifications/src/styled/StyledNotification.ts

```diff
@@ -49,11 +49,10 @@ const colorStyles = (props: INotificationProps & ThemeProps<DefaultTheme>) => {
 /**
  * Supports all `<div>` props
  */
-export const StyledNotification = styled(StyledBase).attrs(props => ({
+export const StyledNotification = styled(StyledBase).attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  role: props.role === undefined ? 'status' : props.role
-}))<INotificationProps>`
+  'data-garden-version': PACKAGE_VERSION
+})<INotificationProps>`
```

Comment:
There was an issue when `tsc` was ran, and `role` was moved to the component as a result.
![Screen Shot 2022-11-07 at 4 06 15 PM](https://user-images.githubusercontent.com/12474067/200417902-65559d35-b325-4060-aba8-c7dbb3220142.png)